templates/homepage/public_service_announcement_partial.html

```diff
@@ -0,0 +1,7 @@
+{% load i18n %}
+<div class="text-center categories col-sm-4">
+    <h2>{% trans "Public Service Announcement" %}</h2>
+    {% if psa_body %}
+        <p>{{ psa_body | safe }}</p>
+    {% endif %}
```

Comment:
Maybe we can add some text if there are no Announcements?

Reply:
There is text in the model that returns when there are no announcements, but it looks like this comment was added before this [one](https://github.com/djangopackages/djangopackages/pull/1047#discussion_r1375429334) so I think this can be resolbed